spec/features/authentication/sign_up_spec.rb

```diff
@@ -26,7 +26,7 @@
     within("form") do
       # Errors show below the inputs, Capybara will show the error as follows
       expect(page).to have_content("Emailcan't be blank")
-      expect(page).to have_content("Passwordcan't be blank")
+      expect(page).to have_content("PasswordMinimum is #{Rails.configuration.devise.password_length.min} characterscan't be blank")
```

Comment:
yeah

`within(email_selector) { expect(page).to have_content("can't be blank") }`